tests/EdgeCaseTest.php

```diff
@@ -15,8 +15,8 @@
 
 final class EdgeCaseTest extends TestCase {
   public function getManualExamples(
-  ): array<(string, string)> {
-    return [
+  ): varray<(string, string)> {
```

Comment:
Thanks; for any future changes, please use vec where it's possible  - I think this will always be the case for dataproviders in unit tests